### PR TITLE
MF-225 Edit Warning Message When Ending a Visit

### PR DESCRIPTION
**Pull request by Gayatri-Kodag**
MF-225 Edit Warning Message When Ending a Visit 

Changed the Warning Message when ending a Visit from "Are you sure to end this visit" to "Are you sure you wish to end this visit?"

### CODE REVIEW

File: src/widgets/visit/visit-button-component.tsx [hunk 1]
```diff
@@ -181,10 +181,12 @@ const closeActiveVisitConfirmation = (currentVisit: any): React.ReactNode => {
   );
 };
 
-const endVisit = (currentVisit: any): React.ReactNode => {
+export const endVisit = (currentVisit: any): React.ReactNode => {
   return (
     <div className={styles.visitPromptContainer} data-testid="end-visit-prompt">
-      <h2>Are you sure to end this visit</h2>
+      <h2 data-testid="end-visit-message">
+        Are you sure you wish to end this visit?
```
Comment (FlorianRappl) [inline src/widgets/visit/visit-button-component.tsx]: Could we make this translatable?

File: src/widgets/visit/visit-button-component.tsx [hunk 2]
```diff
@@ -181,10 +181,12 @@ const closeActiveVisitConfirmation = (currentVisit: any): React.ReactNode => {
   );
 };
 
-const endVisit = (currentVisit: any): React.ReactNode => {
+export const endVisit = (currentVisit: any): React.ReactNode => {
```
Comment (FlorianRappl) [inline src/widgets/visit/visit-button-component.tsx]: Any chance of having a real React component here, e.g.,

```ts
export const EndVisit: React.FC<EndVisitProps> = ({ currentVisit }) => (
  ...
);
```

where

```ts
export interface EndVisitProps {
  currentVisit: VisitData;
}
```

(notice the `VisitData`, I don't know if we have a proper type for this already, but it should not be `any`)

**Review by FlorianRappl — COMMENTED** [review]
I think we are on a great way here. Thanks for the contribution @Gayatri-Kodag !

File: src/widgets/visit/visit-button-component.tsx (hunk 1 above)
Comment (Gayatri-Kodag) [reply src/widgets/visit/visit-button-component.tsx]: Yes, I will make it translatable.

File: src/widgets/visit/visit-button-component.tsx (hunk 2 above)
Comment (Gayatri-Kodag) [reply src/widgets/visit/visit-button-component.tsx]: Yes, I have started working on it to make a React component.

Comment (Gayatri-Kodag) [reply src/widgets/visit/visit-button-component.tsx]: Hey @FlorianRappl , we have Visit for VisitData. I had to change the startDatetime and stopDatetime type from string to Date as it was conflicting between components. I have created a React component for endVisit.

**Review by FlorianRappl — APPROVED** [review]
LGTM! Thanks for the work :beers:

**Pull request merged**